k8s/openebs-pre-release-features.yaml

```diff
@@ -18,10 +18,22 @@ spec:
   # nothing exists at the given path i.e. an empty directory will be created.
   - name: HostPathType
     value: DirectoryOrCreate
+  # SparseDir is a hostPath directory where to look for sparse files
+  - name: SparseDir
+    value: /var/openebs
```

Comment:
Should we make this default as /var/openebs/sparse ?
Reason is that in the default configuration, inside pool pod, /tmp is mounted from /var/openebs/<spc> and /var/openebs will be mounted from /var/openebs and potentially will contain the <spc> folder as well which is not needed.

Reply:
good point! will make this change.